core/src/main/java/org/frankframework/pipes/ChecksumPipe.java

```diff
@@ -25,160 +25,200 @@
 import java.util.zip.Checksum;
 
 import lombok.Getter;
+import org.apache.commons.codec.binary.Base64;
+
 import org.frankframework.configuration.ConfigurationWarning;
 import org.frankframework.core.PipeLineSession;
 import org.frankframework.core.PipeRunException;
 import org.frankframework.core.PipeRunResult;
 import org.frankframework.stream.Message;
 
 /**
- * Pipe to calculate checksum on input.
+ * This pipe can be used to generate a hash for the given message using an algorithm. With this, you can prove integrity of the message. If you
+ * need to prove the authenticity of the message as well, please use the {@link HashPipe} which uses an algorithm and a secret to prove both
+ * integrity and authenticity.
+ * <p>
+ * The hash is generated based on the bytes of the given input message or on the bytes read from the file path if @{inputIsFile} is @{code true}
+ * <p>
+ * The supported algorithms are:
+ * <ul>
+ *     <li>CRC32</li>
+ *     <li>Adler32</li>
+ *     <li>MD5</li>
+ *     <li>SHA</li>
+ *     <li>SHA256</li>
+ *     <li>SHA384</li>
+ *     <li>SHA512</li>
```

Comment:
Als ik dit zo terugzie is het eigenlijk helemaal niet zo lastig om van deze 2 pipes 1 te maken. Misschien toch maandag even in de groep gooien?

Reply:
Sure, ik heb de rest aangepast